scripts/startUserMigration.js

```diff
@@ -0,0 +1,44 @@
+const { program } = require('commander');
+const appPromise = require('../src/app');
+const { error } = require('../src/logger');
+const { schoolModel } = require('../src/services/school/model');
+
+program
+	.option('-s, --systemId <value>', 'id of the system, which will be added to the schools (required argument)')
+	.option(
+		'-S, --schoolIds <value...>',
+		'ids of the schools that should be migrated. If not given all schools with a official school number will be migrated.',
+	);
+
+program.parse();
+
+const options = program.opts();
+
+const startUserMigration = async () => {
+	const { systemId } = options;
```

Comment:
should fail for invalid systemid (we can search for the system and fail when it's not found) and check the system type